plugins/modules/ecs_service_info.py

```diff
@@ -161,6 +168,17 @@ def list_services_with_backoff(self, **kwargs):
     def describe_services_with_backoff(self, **kwargs):
         return self.ecs.describe_services(**kwargs)
 
+    def list_clusters(self):
+        try:
+            paginator = self.ecs.get_paginator("list_clusters")
+            response = paginator.paginate().build_full_result()
+        except (botocore.exceptions.ClientError, botocore.exceptions.BotoCoreError) as e:
+            self.module.fail_json_aws(e, msg="Couldn't list ECS clusters")
+        clusters = list(response["clusterArns"])
+        if not clusters:
+            self.module.fail_json_aws(e, msg="Account does not have any ECS clusters")
```

Comment:
```suggestion
        if not clusters:
            self.module.fail_json_aws(e, msg="Account does not have any ECS clusters")
```

Maybe it's not an error when an account doesn't have any clusters.  
In that case, it should return just an empty array from my point of view.

Reply:
Sure. I'll change the behaviour to this. I'll add an integration test to 10_ecs_cluster.yml before the initial cluster creation.